delivery/src/main/res/layout/list_item_delivery.xml

```diff
@@ -66,17 +65,57 @@
         app:layout_constraintEnd_toStartOf="@id/guideline_name"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/text_item_delivery_time_from"
+        app:layout_goneMarginBottom="@dimen/offset_0dp"
         tools:text="do 3:00" />
 
     <TextView
         android:id="@+id/text_item_delivery_price"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:textAppearance="@style/TextAppearance.Medium"
-        app:layout_constraintBottom_toTopOf="@id/info"
+        app:layout_constraintBottom_toTopOf="@id/autodecline_countdown_text"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="@+id/price_barrier"
         app:layout_constraintTop_toTopOf="parent"
-        tools:text="876.00 Kc" />
+        tools:text="6.00 Kč" />
+
+    <androidx.appcompat.widget.AppCompatImageView
+        android:id="@+id/autodecline_countdown_icon"
+        android:layout_width="@dimen/offset_16dp"
+        android:layout_height="@dimen/offset_16dp"
+        android:src="@drawable/ic_hourglass"
+        app:layout_constraintBottom_toBottomOf="@+id/autodecline_countdown_text"
+        app:layout_constraintHorizontal_chainStyle="packed"
+        app:layout_constraintStart_toStartOf="@+id/price_barrier"
+        app:layout_constraintTop_toTopOf="@+id/autodecline_countdown_text"
+        app:layout_constraintEnd_toStartOf="@+id/autodecline_countdown_text"
+        />
+
+    <TextView
+        android:id="@+id/autodecline_countdown_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="0:08"
+        android:layout_marginStart="@dimen/offset_4dp"
+        android:textAppearance="@style/TextAppearance.Medium"
+        android:textColor="@color/red"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/autodecline_countdown_icon"
+        app:layout_constraintTop_toBottomOf="@+id/text_item_delivery_price"
+        tools:text="0:08" />

```

Comment:
Když by bylo hodnota "30:00:08", tak je to už docela široký. Co ikonu dát nahoru a celý zarovnat dolu nad @+id/info ?